content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -37,27 +32,25 @@ With Netlify, the fundamental building block is a [**Site**](https://docs.netlif
 - [Netlify](https://app.netlify.com/signup) account connected to your GitHub account (Netlify will need access to the repository you will create as part of this guide).
 - [Netlify CLI](https://docs.netlify.com/cli/get-started/) installed.
 - Node.js installed.
-- PostgreSQL CLI `psql` installed.
 
 ## Prisma workflow
 
-Prisma supports different workflows depending on whether you integrate with an existing database or create a new one from scratch. Regardless of the workflow, Prisma relies on the Prisma schema, i.e. `schema.prisma` file.
+Prisma supports different workflows depending on whether you integrate with an existing database or create a new one from scratch. Regardless of the workflow, Prisma relies on the Prisma schema - that is, the `schema.prisma` file.
 
-This guide starts with an empty database created with plain SQL and looks as follows:
+This guide starts with an empty database that you must have already created and looks as follows:
 
-1. Define the database schema using SQL.
-1. Run `prisma db pull` locally which will introspect and populate the `schema.prisma` with models based on the database schema.
-1. Run `prisma generate` which will generate Prisma Client based on the Prisma schema.
+1. Map the existing database schema to your existing database using `prisma migrate dev`.
```

Comment:
I think this would be more Prisma like wording for this process:
```suggestion
1. Migrate the existing Prisma schema to your existing database using `prisma migrate dev`.
```

Reply:
+1, had the same thinking here

Reply:
Ok, I'll make the change.